src/test/java/core/basesyntax/RegistrationServiceImplTest.java

```diff
@@ -0,0 +1,102 @@
+package core.basesyntax;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.junit.jupiter.api.Assertions.assertThrows;
+
+import core.basesyntax.dao.StorageDao;
+import core.basesyntax.dao.StorageDaoImpl;
+import core.basesyntax.db.Storage;
+import core.basesyntax.model.User;
+import core.basesyntax.service.RegistrationService;
+import core.basesyntax.service.RegistrationServiceImpl;
+import org.junit.jupiter.api.AfterEach;
+import org.junit.jupiter.api.BeforeAll;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+
+class RegistrationServiceImplTest {
+    private static User testUser;
+    private static RegistrationService testRegistration;
+
+    @BeforeAll
+    static void atStartOnce() {
+        StorageDao testStorage = new StorageDaoImpl();
+        testRegistration = new RegistrationServiceImpl();
+    }
+
+    @BeforeEach
+    void setUp() {
+        testUser = new User();
+        testUser.setAge(23);
+        testUser.setLogin("SoulARC");
+        testUser.setPassword("qwerty");
+    }
+
+    @AfterEach
+    void clearStorage() {
+        Storage.people.clear();
+    }
+
+    @Test
+    void register_checkingForNonExistentUser_Ok() {
```

Comment:
You have to share the state of a system in this part of the test method name.
It would be enough
```suggestion
    void register_nonExistentUser_ok() {
```